packages/schemas/tables/logs.sql

```diff
@@ -1,3 +1,5 @@
+/* init_order = 2 */
```

Comment:
Hmmm, should we use `/**  */` instead of single asterisk?

Reply:
they have different meanings. `/** */` indicates a JSDoc comment, `/* */` for regular comments

Reply:
Then this would not be removed by `stripLeadingJsDocComments` function, right?

Reply:
right, and it won't be removed i suppose, if you take a look at the regex

Reply:
So.. let's update the regex?

Reply:
no need to remove as it doesn't break anything?